packages/components/src/systems/PharmacySearch/index.tsx

```diff
@@ -16,31 +32,50 @@ export interface PharmacyProps {
 }
 
 export default function PharmacySearch(props: PharmacyProps) {
-  const { store, actions } = PharmacyStore;
+  const client = usePhoton();
   const [selected, setSelected] = createSignal<any>();
   const [query, setQuery] = createSignal('');
+  const [location, setLocation] = createSignal<Location | null>(null);
+  const [pharmacies, setPharmacies] = createSignal<Pharmacy[] | null>(null);
+  const [fetching, setFetching] = createSignal(false);
+  const [openLocationSearch, setOpenLocationSearch] = createSignal(false);
+
+  async function fetchPharmacies() {
+    const { data } = await client!.sdk.apollo.query({
+      query: GetPharmaciesQuery,
+      variables: {
+        location: { latitude: location()?.latitude, longitude: location()?.longitude, radius: 20 }
```

Comment:
Might want to double check that if you don't pass in a radius it defaults to something in the backend